common/components/meta-list/template.njk

```diff
@@ -1,7 +1,7 @@
 <dl class="app-meta-list {% if params.classes -%} {{ params.classes }}{%- endif %}">
   {% for item in params.items %}
     {% if item.value.text or item.value.html %}
-      <div class="app-meta-list__item">
+      <div class="app-meta-list__item app-!-position-relative">
```

Comment:
Could this be added using `params.classes` rather than this class applied and style being applied to all areas where this component is used.

Reply:
Made redundant by updating the component styling